src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ReferralController.kt

```diff
@@ -59,6 +62,7 @@ class ReferralController(
     )
   }
 
+  @JsonView(Views.SentReferral::class)
   @PostMapping("/draft-referral/{id}/send")
   fun sendDraftReferral(@PathVariable id: UUID, authentication: JwtAuthenticationToken): ResponseEntity<SentReferralDTO> {
```

Comment:
🤔 Would be nice if the `SentReferralDTO` could be annotated with `JsonView`, adding default behaviour to it on serialisation

Reply:
yes i tried this but with no joy...

Reply:
ah well. annotations 🙃